schemas/test/en/test_currency.json

```diff
@@ -93,10 +93,93 @@
+                                            "value": -1000000
+                                        }
+                                    },
+                                    {
+                                        "id": "negative-answer-usd",
+                                        "label": "How much did you spend?",
+                                        "mandatory": false,
+                                        "type": "Currency",
+                                        "currency": "USD",
+                                        "decimal_places": 2,
+                                        "maximum": {
+                                            "value": 10000
+                                        },
+                                        "minimum": {
+                                            "value": -1000000
+                                        }
+                                    },
+                                    {
+                                        "id": "negative-answer-eur",
+                                        "label": "How much did you spend?",
+                                        "mandatory": false,
+                                        "type": "Currency",
+                                        "currency": "EUR",
+                                        "decimal_places": 2,
+                                        "maximum": {
+                                            "value": 10000
+                                        },
+                                        "minimum": {
+                                            "value": -1000000
+                                        }
+                                    },
+                                    {
+                                        "id": "negative-answer-jpy",
+                                        "label": "How much did you spend?",
+                                        "mandatory": false,
+                                        "type": "Currency",
+                                        "currency": "JPY",
+                                        "maximum": {
+                                            "value": 1000000
+                                        },
+                                        "minimum": {
+                                            "value": -1000000
+                                        }
+                                    },
+                                    {
+                                        "id": "negative-answer-min-max-range",
+                                        "label": "How much did you spend? (Max range)",
+                                        "mandatory": false,
+                                        "type": "Currency",
+                                        "currency": "GBP",
+                                        "minimum": {
+                                            "value": -999999999999
```

Comment:
Error message displayed for this doesn't seem right
<img width="1678" alt="Screenshot 2023-04-18 at 11 43 43" src="https://user-images.githubusercontent.com/108286538/232753676-e9c0da15-26c3-4bdd-a9d1-bd360bc56e37.png">

Reply:
Updated should show the correct error message for all the times you exceed the minimum boundary now in any box. 👍